run_octane.c

```diff
@@ -0,0 +1,90 @@
+#include "quickjs/quickjs.h"
+#include "quickjs/quickjs-libc.h"
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <time.h>
+
+double total = 0;
+
+int Execute(JSContext *ctx, const char *filename, int eval_flags) {
+    FILE *file = fopen(filename, "r");
+
+    if (file == NULL) {
+        return 1;
+    }
+
+    fseek(file, 0, SEEK_END);
+    long file_size = ftell(file);
+    fseek(file, 0, SEEK_SET);
+
+    char *buffer = (char *)malloc(file_size + 1);
+    if (buffer == NULL) {
+        fclose(file);
+        fprintf(stderr, "Memory allocation error\n");
+        return 1;
+    }
+
+    size_t bytesRead = fread(buffer, 1, file_size, file);
```

Comment:
It's easier if you write this like so:
```suggestion
    if (1 != fread(buffer, file_size, 1, file)) {
```
Because then fread either reads everything, or nothing.